src/main/java/org/broadinstitute/hellbender/tools/sv/SVCallRecordUtils.java

```diff
@@ -0,0 +1,467 @@
+        if (!record.getType().equals(StructuralVariantType.INV)) {
+            return Stream.of(record);
+        }
+        Utils.validateArg(record.isIntrachromosomal(), "Inversion " + record.getId() + " is not intrachromosomal");
+        final SVCallRecord positiveBreakend = new SVCallRecord(record.getId(), record.getContigA(),
+                record.getPositionA(), true, record.getContigB(), record.getPositionB(), true, StructuralVariantType.BND, null,
+                record.getAlgorithms(), record.getAlleles(), record.getGenotypes(), record.getAttributes());
+        final SVCallRecord negativeBreakend = new SVCallRecord(record.getId(), record.getContigA(),
+                record.getPositionA(), false, record.getContigB(), record.getPositionB(), false, StructuralVariantType.BND, null,
+                record.getAlgorithms(), record.getAlleles(), record.getGenotypes(), record.getAttributes());
+        return Stream.of(positiveBreakend, negativeBreakend);
+    }
+
+    /**
+     * Creates a new {@link SVCallRecord} from the given {@link VariantContext}, keeping any variant fields.
+     * @see SVCallRecordUtils#create(VariantContext, boolean)
+     */
+    public static SVCallRecord create(final VariantContext variant) {
+        return create(variant, true);
+    }
+
+    /**
+     * Creates a new {@link SVCallRecord} from the given {@link VariantContext}.
+     * @param variant SV record
+     * @param keepVariantAttributes retain variant attribute fields
+     * @return converted record
+     */
+    public static SVCallRecord create(final VariantContext variant, boolean keepVariantAttributes) {
+        Utils.nonNull(variant);
+        final String id = variant.getID();
+        final String contigA = variant.getContig();
+        final int positionA = variant.getStart();
+
+        final StructuralVariantType type = inferStructuralVariantType(variant);
+        final List<String> algorithms = getAlgorithms(variant);
+
+        final String strands;
+        if (type.equals(StructuralVariantType.DEL) || type.equals(StructuralVariantType.INS) ||
+                type.equals(StructuralVariantType.CNV) || type.equals(StructuralVariantType.DUP)) {
+            // SVCallRecord class can resolve these
+            strands = null;
+        } else {
+            strands = getStrands(variant, type);
+        }
+        final Boolean strand1 = strands == null ? null : strands.startsWith(SVCallRecord.STRAND_PLUS);
+        final Boolean strand2 = strands == null ? null : strands.endsWith(SVCallRecord.STRAND_PLUS);
+
+        final Integer length;
+        if (type.equals(StructuralVariantType.BND) || type.equals(StructuralVariantType.DEL) ||
+                type.equals(StructuralVariantType.DUP) || type.equals(StructuralVariantType.CNV) ||
+                type.equals(StructuralVariantType.INV)) {
+            // SVCallRecord class can resolve these
+            length = null;
```

Comment:
Hm which comment was that? The list is too long I can't find it!

I addressed a minor comment to move this logic out of the `getLength()` method. It's difficult to handle this elegantly because SVLEN is sometimes, but not always, redundant (i.e. with CNVs), whereas with insertions it is actually additional information (or even unknown).